<launch>
    <node pkg="mtts_can_ros" type="predict_ros.py" name="mtts_hr" output="screen">
        <rosparam file="$(find mtts_can_ros)/config/params.yaml"/>
        <param name="path" value="$(find mtts_can_ros)/models/mtts_can.hdf5"/>
        <remap from="/image" to="/camera/image_color"/>
    </node>
</launch>
